packages/ra-ui-materialui/src/preferences/Configurable.stories.tsx

```diff
@@ -0,0 +1,203 @@
+            borderRadius={3}
+            p={1}
+            width={300}
+            bgcolor={color}
+        >
+            <Typography variant="h6">Lorem ipsum</Typography>
+            <Typography>
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit. Qui,
+                quia rem? Nulla asperiores ea beatae iure, dignissimos ut
+                perferendis nemo reiciendis reprehenderit, consequuntur debitis
+                maiores! Quaerat dolor unde dolorum qui.
+            </Typography>
+            {children}
+        </Box>
+    );
+};
+
+const TextBlockEditor = () => {
+    const preferenceKey = usePreferenceKey();
+    const [color, setColor] = useStore(`${preferenceKey}.color`, '#ffffff');
+    useSetInspectorTitle('ra.inspector.textBlock', { _: 'Text block' });
+    return (
+        <div>
+            <label htmlFor="color">Background color</label>
+            {/* uncontrolled component */}
+            <input
+                defaultValue={color}
+                onBlur={e => setColor(e.target.value)}
+                id="color"
+            />
+        </div>
+    );
+};
+
+const ConfigurableTextBlock = ({
+    preferenceKey = 'textBlock',
+    ...props
+}: any) => (
+    <Configurable editor={<TextBlockEditor />} preferenceKey={preferenceKey}>
+        <TextBlock {...props} />
+    </Configurable>
+);
+
+const SalesBlock = () => {
+    const preferenceKey = usePreferenceKey();
+    const [showDate] = useStore(`${preferenceKey}.showDate`, true);
+    return (
+        <Box
+            display="flex"
+            border="solid 1px lightgrey"
+            borderRadius={3}
+            p={1}
+            width={200}
+        >
+            <Box flex="1" mr={1}>
+                <Typography variant="h6">Sales</Typography>
+                {showDate && <Typography variant="caption">Today</Typography>}
+                <Typography variant="h4" textAlign="right" mt={2}>
+                    $4,452
+                </Typography>
+            </Box>
+            <Box bgcolor="lightgrey" display="flex" alignItems="center" p={1}>
+                <TimelineIcon />
+            </Box>
+        </Box>
+    );
+};
+
+const SalesBlockEditor = () => {
+    const preferenceKey = usePreferenceKey();
+    const [showDate, setShowDate] = useStore(`${preferenceKey}.showDate`, true);
+    useSetInspectorTitle('ra.inspector.salesBlock', { _: 'Sales block' });
+    return (
+        <>
+            <label htmlFor="showDate">Show date</label>
+
+            <input
+                type="checkbox"
+                defaultChecked={showDate}
+                onChange={e => setShowDate(showDatz => !showDate)}
+                id="showDate"
+            />
+        </>
+    );
+};
+
+const ConfigurableSalesBlock = ({ preferenceKey = 'salesBlock', ...props }) => (
+    <Configurable editor={<SalesBlockEditor />} preferenceKey={preferenceKey}>
+        <SalesBlock {...props} />
+    </Configurable>
+);
+
+export const Basic = () => (
```

Comment:
The Basic story doesn't correctly show the Inspector's title and content

Reply:
You mean it's not translated? That's normal - the i18n story is there to show the default translation

Reply:
Yes, that was what I meant. Ok then.